cfg/std.cfg

```diff
@@ -7519,6 +7519,32 @@ initializer list (7) string& replace (const_iterator i1, const_iterator i2, init
       <not-uninit/>
     </arg>
   </function>
+  <!-- template< class ForwardIt > ForwardIt std::next(ForwardIt it, typename std::iterator_traits<ForwardIt>::difference_type n = 1 ); // since C++11 until C++17 -->
+  <!-- template< class InputIt > constexpr InputIt std::next(InputIt it, typename std::iterator_traits<InputIt>::difference_type n = 1 ); // since C++17 -->
+  <function name="std::next">
+    <noreturn>false</noreturn>
+    <returnValue type="iterator" container="1"/>
+    <use-retval/>
+    <arg nr="1">
+      <iterator container="1" type="middle"/>
```

Comment:
@danmar Does `<returnValue type="iterator" container="1"/>` and `<iterator container="1" type="middle"/>` make any sense here? I have not found a comparable function configuration. `first` and `last` for the iterator type do not make sense i guess. But there must be a type specified.

Reply:
sorry... off the top of my head I am not sure about the proper `returnValue` configuration. But I guess the `iterator` configuration is good.

Reply:
Shall I remove the `returnValue` and `iterator` configuration to be safe?

Reply:
yes that sounds good.. leaving some TODO comment would not hurt.

Reply:
Ok, i added TODO comments